features/steps/helpers.py

```diff
@@ -0,0 +1,49 @@
+from time import sleep
+
+from selenium.common.exceptions import ElementClickInterceptedException
+from selenium.common.exceptions import ElementNotInteractableException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+def sign_in(context, login=None, browser=None):
+    if login is None:
+        login = context.username
+    if browser is None:
+        browser = context.browser
+    browser.get(context.base_url)
+    login_form = WebDriverWait(browser, 10).until(
+        expected_conditions.visibility_of_element_located((By.CLASS_NAME, "login"))
+    )
+    cautious_send_keys(browser=login_form, xpath=".//input[@id='username']", sequence=login)
+    password = login_form.find_element_by_xpath(".//input[@id='password']")
+    password.send_keys("userpasswd")
+    submit_button = login_form.find_element_by_xpath(".//input[@type='submit']")
+    submit_button.click()
+
+
+def cautious_click(browser, xpath, iterations=5, timeout=2):
+    iterations = iterations - 1
+    element = browser.find_element_by_xpath(xpath)
+    try:
+        sleep(timeout)
+        element.click()
+    except ElementClickInterceptedException as Error:
```

Comment:
That's the best thing I could come up with to keep this thing somewhat deterministic. Seems to work alright, as long as there's a sleep before first action (otherwise it would fail radomnly, I'm not sure why, _no comprendo el frontendo_).